### PR TITLE
debian: make mkversion.sh fall back to looking at the changelog

### PR DESCRIPTION
**Pull request by chipaca**
this can happen if e.g. no git, or if it's a git export.
Also make it more verbose: mkversion.sh will now print e.g.
`*** Setting version to '2.0.8' from debian/changelog.`
to stderr.

### CODE REVIEW

**Comment by fgimenez** [comment]
retest this please

File: mkversion.sh [hunk 1]
```diff
@@ -6,7 +6,21 @@ if [ -z "$GOPACKAGE" ]; then
...
 fi
 
-v="$( git describe --dirty --always | sed -e 's/-/+git/;y/-/./' )"
+if which git >/dev/null; then
+    v="$( git describe --dirty --always | sed -e 's/-/+git/;y/-/./' )"
+    o=git
+fi
+
+if [ -z "$v" ]; then
+    v="$(head -n1 "$(dirname $0)"/debian/changelog | sed -e 's/^.*(\(.*\)).*/\1/')"
```
Comment (mvo5) [inline mkversion.sh]: You could use `dpkg-parsechangelog --show-field Version` here I think.

**Comment by mvo5** [comment]
Nice, thank you. One suggestion, otherwise good.

File: mkversion.sh (hunk 1 above)
Comment (chipaca) [inline mkversion.sh]: Good one! I looked around for exactly that and didn't find it. Fixed.

**Comment by zyga** [comment]
Looks good, +1

**Comment by chipaca** [comment]
retest this please

**Pull request merged**